libopenage/testing/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 add_sources(libopenage
-	testing.cpp
+	"testing.cpp"
+	"benchmark_test.cpp"
```

Comment:
cmake and I have a long history of hate and shouting - I always do quotes because of it.